### ISSUE TITLE
Branding

### ISSUE THREAD

**Issue by arianesoft** [PowerDNS/pdns#5564]
•Program: Authoritative
•Issue type: Feature request

**# Description**

I ‘d like to customize it, change the title at the top of the page for example, but I can see that this title is written explicitely in every page.

Would it be possible for you to put that title into a separate file that would be included in the code?
So I would only have to change the include file content.

**Comment by ahupowerdns** [comment]
to clarify, you are talking about the built-in authoritative webserver?

**Comment by arianesoft** [comment]
exactly, I'd like to easily customize the authoritative webserver interface without changing the source code.
